configs/babylon-integration/consumer-fpd.conf

```diff
@@ -31,7 +31,7 @@ MaxSubmissionRetries = 100
 FastSyncInterval = 20s
 
 ; The maximum number of blocks to catch up for each fast sync
-FastSyncLimit = 10
+FastSyncLimit = 100
```

Comment:
it causes confusion to have same changes in two PRs (https://github.com/Snapchain/babylon-deployment/pull/40)

we should either stack them or don't change same thing at two PRs

Reply:
yes, it makes senses.

 i added it only for testing changes with one branch.